server/channels/api4/command_test.go

```diff
@@ -432,17 +437,21 @@ func TestListAutocompleteCommands(t *testing.T) {
 	})
 
 	t.Run("NoMember", func(t *testing.T) {
-		client.Logout(context.Background())
+		_, err := client.Logout(context.Background())
+		require.NoError(t, err)
+
 		user := th.CreateUser()
-		th.SystemAdminClient.RemoveTeamMember(context.Background(), th.BasicTeam.Id, user.Id)
-		client.Login(context.Background(), user.Email, user.Password)
```

Comment:
This call would error as a newly created user is not part of `BasicTeam`.

Reply:
Great catch! Advantage of catching errors.